tests/network/localnet/conftest.py

```diff
@@ -256,3 +258,15 @@ def localnet_ovs_bridge_client(
     ) as client:
         assert client.is_running()
         yield client
+
+
+@pytest.fixture()
+def migrated_localnet_vm(
+    server_client_localnet_vms: tuple[BaseVirtualMachine, BaseVirtualMachine],
+) -> BaseVirtualMachine:
+    server_vm, client_vm = server_client_localnet_vms
+    with client_server_active_connection(client_vm=client_vm, server_vm=server_vm):
+        pass
+
+    migrate_vm_and_verify(vm=client_vm)  # type: ignore[no-untyped-call]
```

Comment:
IIUC, this flow means that the TCP connection remains active during the migration.
But the purpose of this test ([CNV-11925](https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-11925)) is to verify that new connection can be obtained after the migration.
The reason for initiating a TCP connection before the migration is to make sure that TCP is possible between the VMs, and once this is verified - the flow should continue to migrating and starting a new connection after that.

Reply:
Once `client_server_active_connection` finishes (it has an empty body), the TCP connection is disconnected. When reaching `migrate_vm_and_verify` there is no active TCP connection.